src/test/java/net/openhft/chronicle/queue/IndexForIDTest.java

```diff
@@ -144,7 +144,7 @@ private void copy(String fromID, TimeSetter setTime, String toID) {
 
     @Test //(timeout = 10_000)
     public void staged() {
-        expectException("This functionality has been deprecated and in future will only be available in Chronicle Queue Enterprise");
+        ignoreException("This functionality has been deprecated and in future will only be available in Chronicle Queue Enterprise");
```

Comment:
For some reason this didn't occur in the Mac build so I had to ignore instead of expect